src/components/ScoreBox/WordScoreList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import TotalScore from './TotalScore'
+const WordScoreList = (props) => {
+  const {words, scores, totalScore}  = props
+
+  const wordsList = words.map(function(word, index) {
+    return (<li key={word.toString()+index}>{word}</li>)
```

Comment:
yes because 
<div className="words">
    <h2>WORD</h2>
    {wordsList}
</div>

so that wordsList will be array and requires key or it will give you lots of warnings.

It won't make any difference in performance because the way key is generated (using index)